python/doc/theory/meta_modeling/enumeration_strategy.rst

```diff
@@ -3,58 +3,115 @@
-  strategy for enumerating the infinite PC series has to be defined.
-  This is the scope of the current section.
+The *functional chaos* expansion allows one to obtain an explicit
+representation of the random response :math:`\underline{Y}` of the
+model under consideration. More precisely, the response is cast
+as a converging series featuring an orthonormal basis. For
+computational purpose, it is necessary though to retain a finite
+number of terms by truncating the expansion. First of all, a specific
+strategy for enumerating the infinite PC series has to be defined.
+This is the scope of the current section.
 
 Given an input random vector :math:`\vect{X}` with prescribed
 probability density function (PDF) :math:`f_{\vect{X}}(\vect{x})`, it is
-possible to build up a *polynomial chaos* (PC) basis
-:math:`\{\psi_{\idx},\idx \in \NM\}` . Of interest is the definition of
-enumeration strategies for exploring this basis, i.e. of suitable
-*enumeration functions* :math:`\tau` from :math:`\Nset` to :math:`\NM`,
-which creates a one-to-one mapping between an integer :math:`j` and a
-multi-index :math:`\idx`.
+possible to build up a *functional chaos* multivariate basis
+:math:`\{\psi_{\idx},\idx \in \NM\}` by tensorization of univariate
+functions associated to each marginal input variable.
 
-Linear enumeration strategy
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We define the multi-indices associated to the marginal degrees of the
+univariate marginal functions:
 
-Let us first define the *total degree* of any multi-index :math:`\idx`
-in :math:`\NM` by :math:`\sum_{i=1}^{n_X} \alpha_i`. A natural choice to
-sort the PC basis (i.e. the multi-indices :math:`\idx`) is the
-lexicographical order with a constraint of increasing total degree.
-Mathematically speaking, a bijective enumeration function :math:`\tau`
-is defined by:
+.. math::
+
+    \idx = (\alpha_1, \dots, \alpha_{n_X}) \in \mathbb{N}^{n_X}
+
+The component :math:`\alpha_i` defines the marginal degree of the univariate
+basis term associated to the variable :math:`X_i`.
+For :math:`i = 1, ..., n_X`, let :math:`\pi_{\alpha_i}^{(i)}` be a univariate basis term of marginal degree :math:`\alpha_i` depending on the i-th standardized variable :math:`\xi_i`.
+Then, the multivariate basis term :math:`\psi_{\idx}` is defined by the product:
+
+.. math::
+
+    \psi_{\idx} (\vect{\xi}) = \pi_{\alpha_1}^{(1)}(\xi_1) \times \dots \times \pi_{\alpha_{n_X}}^{({n_X})}(\xi_{n_X})
+
+for any :math:`\vect{\xi} \in \mathbb{R}^{n_X}`.
+
+Let us first define the *length* of any multi-index :math:`{\idx} \in {\Nset}^{n_X}` by
+
+.. math::
+
+    |{\idx}| = \sum_{i=1}^{n_X} \alpha_i
+
+When the multi-index represents the marginal degrees of a polynomial, 
+the length of the multi-index is the total degree of the polynomial. 
+
+An enumeration rule is a method to explore this basis. 
+It is defined by an enumeration function :math:`\tau` from :math:`\Nset` to :math:`\NM`,
```

Comment:
Isn't this function called \idx afterwards? This is unfortunate because it creates confusion between an element of {\Nset}^{n_X} and a function from \Nset to {\Nset}^{n_X}.

My recommendation would be to consistently use \tau for the function, and maybe write it \vect{\tau} so we understand the output has dimension >1.

Reply:
indeed, lets used tau